libdnf/hy-iutil-private.hpp

```diff
@@ -25,12 +25,28 @@
 #include "hy-types.h"
 #include "sack/packageset.hpp"
 
+// Use 5 bytes to be future proof even though the version currently is only "1.2"
+#define LIBSOLV_SOLVFILE_VERSION_BYTES 8
+#define DNF_SOLVFILE_VERSION "01.0"
+#define DNF_SOLVFILE_VERSION_BYTES 4
+#define DNF_SOLVFILE_MAGIC "SDNF"
```

Comment:
Not sure if I chose good values for DNF_SOLVFILE_VERSION and DNF_SOLVFILE_MAGIC.